docs/sphinx_source/guide/buyers-guide.md

```diff
@@ -0,0 +1,41 @@
+# Buyer's Guide
+
+## Quantitative phase imaging:
+
+You can use a transmitted light source (LED or a lamp) and a condenser commonly available on almost all microscopes. In addition to the transmitted light imaging path, you will need a motorized stage for acquiring through-focus image stacks.
+
+## Quantitative polarization imaging (PolScope):
+
+The following list of components assumes that you already have a transmitted light source (LED or a lamp) and a condenser.
+
+Buyers have two options:
+1. buy a complete hardware kit from the OpenPolScope project, or
+2. assemble your own kit piece by piece.
+
+### Buy a kit from the OpenPolScope project
+
+- Read about the [OpenPolScope Hardware Kit](https://openpolscope.org/pages/OPS_Hardware.htm).
+- Complete the [OpenPolScope information request form](https://openpolscope.org/pages/Info_Request_Form.htm).
+
+### Buy individual components
+
+The components are listed in the order in which they process light. See the build video here to see how to assemble these components on your microscope.
+
+<video>
+   <source src="https://github.com/user-attachments/assets/a0a8bffb-bf81-4401-9ace-3b4955436b57" type="video/mp4">
+</video>
```

Comment:
Can we embed this like the video clips like on the README, it's too cool to not be viewable :(

https://github.com/user-attachments/assets/9c177f9b-b0ec-4034-9aab-f026f29669bc